### DISCUSSION TITLE
Single-language book format (not bilingual)

### DISCUSSION THREAD

**Discussion by gcharycki** [yihong0618/bilingual_book_maker#410] [Q&A]
I would like to translate an EPUB file, for example, from English to French, but I do not want the new file to have the text repeated twice; I want it to appear only once (in French). I would like every word to be in French, not just the content, but also the book title, chapters, etc. How can I achieve this? Unfortunately, I cannot find the required option.

**Comment by yihong0618** [comment]
--single_translate 

args

**Comment by gcharycki** [comment]
I still get this kind of irregular mashup.

![image](https://github.com/yihong0618/bilingual_book_maker/assets/72350476/2010be0c-4922-4dea-bd1a-c5c09afd15e4)

For that command:

py make_book.py --book_name test_books/animal_farm.epub --openai_key XXX  --test --single_translate --language pl
